src/IssueViz/Commands/IssueVizPackage.vsct

```diff
@@ -22,6 +22,15 @@
         </Strings>
       </Button>
 
+      <Button guid="guidIssueVizPackageCmdSet" id="IssueVisualizationToolWindowErrorListCommandId" priority="0x0200" type="Button">
+        <Parent guid="guidDaemonCmdSet" id="grpDaemonErrorList" />
+        <CommandFlag>DynamicVisibility</CommandFlag>
+        <CommandFlag>DefaultInvisible</CommandFlag>
+        <Strings>
+          <ButtonText>SonarLint Issue Visualization</ButtonText>
```

Comment:
I'm not sure about the text label for the menu item. It's ok for the tool window menu since it's similar to the other commands on the same menu, but it's not as meaningful in this context. I'm not sure would be better - `Show SonarLint Issue Visualization`?